makefiles/Base.toml

```diff
@@ -35,12 +70,8 @@ which musl-gcc 2> /dev/null > /dev/null || {
     echo "Install musl-tools on your system!";
     exit 1
 }
-rustup target add "$CARGO_MAKE_RELEASE_FLOW_TARGET"
```

Comment:
ditto

Reply:
Here, we were the only ones assigning CARGO_MAKE_RELEASE_FLOW_TARGET, and it was with a constant value, so I simply substituted the constant into the  ENV variable.